### PR TITLE
Customized bottom tab bar

### PR DESCRIPTION
**Pull request by EmmanuelWLX**
## Summary
- Customized tab bar to make it match with invision

### Android
- Nexus 6p
bottomTab-nexus6p-1
bottomTab-nexus6p-2

### iOS
- iPhone 6
bottomTab-iPhone6-1
bottomTab-iPhone6-2

## Trello card
https://trello.com/c/Qzth7sts/18-n-74-tab-bar

### CODE REVIEW

File: LibraryApp/src/app/AppContainer.js [hunk 1]
```diff
@@ -30,4 +35,36 @@ const BooksNavigator = createStackNavigator(
   }
 );
 
-export default createAppContainer(BooksNavigator);
+const TabNavigator = createBottomTabNavigator(
+  {
```
Comment (mmamaniwolox) [inline LibraryApp/src/app/AppContainer.js]: extract this as a constant

File: LibraryApp/src/app/AppContainer.js [hunk 2]
```diff
@@ -30,4 +35,36 @@ const BooksNavigator = createStackNavigator(
...
+    [Routes.Library]: BooksNavigator,
+    [Routes.EmptyScreen]: EmptyScreen
+  },
+  {
+    defaultNavigationOptions: ({ navigation }) => ({
```
Comment (mmamaniwolox) [inline LibraryApp/src/app/AppContainer.js]: this can be extracted too

File: LibraryApp/src/app/screens/EmptyScreen/index.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { View } from 'react-native';
+
+import styles from './styles';
+
+function EmptyScreen() {
```
Comment (mmamaniwolox) [inline LibraryApp/src/app/screens/EmptyScreen/index.js]: whats this for?

File: LibraryApp/src/app/screens/EmptyScreen/styles.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+import { StyleSheet } from 'react-native';
+
+export default StyleSheet.create({
+  container: {}
```
Comment (mmamaniwolox) [inline LibraryApp/src/app/screens/EmptyScreen/styles.js]: ???

File: LibraryApp/src/app/screens/EmptyScreen/index.js (hunk 3 above)
Comment (EmmanuelWLX) [reply LibraryApp/src/app/screens/EmptyScreen/index.js]: at that point the card said that it should navigate to an empty screen, and I took it very literal 😂

**Pull request merged**